python/lib/dependabot/python/update_checker/pip_compile_version_resolver.rb

```diff
@@ -25,9 +25,9 @@ class UpdateChecker
       # rubocop:disable Metrics/ClassLength
       class PipCompileVersionResolver
         GIT_DEPENDENCY_UNREACHABLE_REGEX =
-          /git clone --filter=blob:none -q (?<url>[^\s]+).* /.freeze
+          /git clone --filter=blob:none --quiet (?<url>[^\s]+).* /.freeze
         GIT_REFERENCE_NOT_FOUND_REGEX =
-          /egg=(?<name>\S+).*.*WARNING: Did not find branch or tag \'(?<tag>[^\n"]+)\'/m.freeze
+          /WARNING:pip\._internal\.vcs\.git:Did not find branch or tag '(?<tag>[^\n"]+)'/m.freeze
```

Comment:
Curious to see an example of this log message. Is there really no horizontal whitespace after the colons?

Reply:
Huh, weird. Not the kind of default formatting I'd expect for a logging class.

We could try to make this less fragile by focusing on the message itself (`Did not find branch or tag 'v15.1.2', assuming revision or ref.` found [here](https://github.com/pypa/pip/blob/b2b7aefe2f4a40fb34039adaab39897e80df687c/src/pip/_internal/vcs/git.py#L228)). But I'm happy to go with this if it works.

Reply:
Good suggestion, we don't need to be checking the path to that log as well.